packages/gn/build.sh

```diff
@@ -4,8 +4,8 @@ TERMUX_PKG_LICENSE="BSD 3-Clause"
 TERMUX_PKG_MAINTAINER="Yaksh Bariya <yakshbari4@gmail.com>"
 # While updating commit hash here also update it in
 # termux_setup_gn
-_COMMIT=69ec4fca1fa69ddadae13f9e6b7507efa0675263
-TERMUX_PKG_VERSION=0.1
+_COMMIT=46b572ce4ceedfe57f4f84051bd7da624c98bf01
+TERMUX_PKG_VERSION=0.2
```

Comment:
Suggesting to change "version" to date.

Reply:
Sure, but should I also add a epoch?

Reply:
In this case no need. Both dates formats `20210909` and `2021.09.09` represent version "newer" than `0.1`.